src/tersets.zig

```diff
@@ -92,21 +91,23 @@ pub fn decompress(
 
     const method_index: u8 = compressed_values[compressed_values.len - 1];
 
-    if (method_index > getMaxMethodIndex(Method)) return Error.UnknownCompression;
+    if (method_index > getMaxMethodIndex()) return Error.UnknownMethod;
 
     const method: Method = @enumFromInt(method_index);
+    const compressed_values_slide = compressed_values[0 .. compressed_values.len - 1];
```

Comment:
Maybe change `_slide` to [`_slice`](https://ziglang.org/documentation/master/#Slices).

Reply:
Definitely, that was the initial intention.